src/main/java/frc/robot/commands/ShootCommand.java

```diff
@@ -6,17 +6,19 @@
 
 import edu.wpi.first.wpilibj.Timer;
 import edu.wpi.first.wpilibj2.command.Command;
-import frc.robot.subsystems.Shooter;
-// import frc.robot.subsystens.Intake; TODO Coordinate with intake to import the right subsystem
+import frc.robot.subsystems.*;
 
 public class ShootCommand extends Command {
   private final Shooter shooter;
+  private final Intake intake;
   private double speed = 1; // TODO Change to intake motor speed (in rotations per second)
   private Timer timer = new Timer();
 
-  public ShootCommand(Shooter shooter) {
+  public ShootCommand(Shooter shooter, Intake intake) {
     this.shooter = shooter;
+    this.intake = intake;
     addRequirements(shooter);
+    addRequirements(intake);
```

Comment:
`addRequirements()` should be a variadic method which means you should be able to call it like `addRequirements(shooter, intake)`